### PR TITLE
gucharmap 3.16.2 (new formula)

### PR DESCRIPTION
**Pull request by tschoonj**
plus its dependency desktop-file-utils

### CODE REVIEW

File: Library/Formula/desktop-file-utils.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}",
+                          "--with-lispdir=#{share}/emacs/site-lisp/desktop-file-utils"
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/update-desktop-database", "--help"
```
Comment (bfontaine) [inline Library/Formula/desktop-file-utils.rb]: Could you please write a test that does more than just `--help` or `--version`?

**Comment by tschoonj** [comment]
@bfontaine Done!

**Comment by tschoonj** [comment]
@bfontaine @DomT4 Could you merge this Formula in please?

Thanks!

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**